runtimes/elixir-1.14/server/test/server_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule ServerTest do
```

Comment:
Is this unit test for HTTP server? If so, I think we can remove this. Our tests system should have good enough coverage.